dev/breeze/src/airflow_breeze/commands/release_management_commands.py

```diff
@@ -352,7 +378,10 @@ def prepare_airflow_packages(
     # Copy all files in the dist directory in container to the host dist directory (note '/.' in SRC)
     run_command(["docker", "cp", f"{container_id}:/opt/airflow/dist/.", "./dist"], check=True)
     run_command(["docker", "rm", "--force", container_id], check=True)
-    get_console().print("[success]Successfully prepared Airflow package!\n\n")
+    get_console().print("[success]Successfully prepared Airflow packages:")
```

Comment:
Let us have a different log from: https://github.com/apache/airflow/pull/36744/files#diff-54ea68b5e1e61ac0437df0ab7d99dc508595323b11847a957931e0d9007962d7R319 for easier tracing when it fails?

Reply:
Not sure I I know what you are asking for :)?